astronomer/providers/amazon/aws/triggers/emr.py

```diff
@@ -61,14 +61,11 @@ async def run(self) -> AsyncIterator["TriggerEvent"]:  # type: ignore[override]
                 elif query_status in ("FAILED", "CANCELLED", "CANCEL_PENDING"):
                     msg = f"EMR Containers sensor failed {query_status}"
                     yield TriggerEvent({"status": "error", "message": msg})
-                    return
                 else:
                     msg = "EMR Containers sensors completed"
                     yield TriggerEvent({"status": "success", "message": msg})
-                    return
```

Comment:
Can you please check that integration test dont break post this change

Reply:
I tested for one dag `redshift_cluster_management` will run once all